cluster-up.sh

```diff
@@ -27,5 +27,24 @@ vagrant ssh k8s-master -c "grep 'kubeadm join' /tmp/kubeadm-init" > kubeadm-join
 cat kubeadm-join | xargs -I {} vagrant ssh k8s-worker-0 -c "sudo {}"
 cat kubeadm-join | xargs -I {} vagrant ssh k8s-worker-1 -c "sudo {}"
 
+# Getting setup for kubectl on host machine
+# copy this to ~/.kube/ and kubectl should work
+vagrant ssh k8s-master -c "sudo cat /etc/kubernetes/admin.conf" > config
+
+echo "
+### Important 
+config file created to match cluster. Please copy to ~/.kube/ folder for kubectl to work
+"
+
 # Clean up
 rm -fv kubeadm-join
+
+# Alert user about routing
+echo "
+This is added to each hosts /etc/hosts:
+192.168.50.0 lb1
```

Comment:
The IP address and hostname is not consistent with the change in `install-kubeadm.sh` below.
Also, in the `Vagrantfile`, the load balancer VM is not actually created.